src/sentry/static/sentry/less/layout.less

```diff
@@ -1,5 +1,8 @@
 html {
   height: 100%;
+  font-family: sans-serif;
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
```

Comment:
This file is a nightmare and these markup styles should be refactored into base. Ignoring for later.